decidim/decidim
# Refactor intiative wizard

#### :tophat: What? Why?
This PR improves the Create initiative wizard, by removing the session support in favor of the database support.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #5128
- Fixes #6264
- Fixes #9845

#### Testing
1. Visit the admin panel , and enable the WYSIWYG for public participants
2. Configure the initiatives
3. Create an initiative. 
4. See that you get an error 
5. Apply patch 
6. Create an initiative 
7. See there is no error. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
